auto_update_tests.py

```diff
@@ -258,76 +253,67 @@ def update_binaryen_js_tests():
 
     print('\n[ checking binaryen.js testcases... ]\n')
     node_has_wasm = NODEJS and node_has_webassembly(NODEJS)
-    for s in sorted(os.listdir(os.path.join(options.binaryen_test, 'binaryen.js'))):
-        if not s.endswith('.js'):
-            continue
-        print(s)
+    for s in get_tests(BINARYEN_JS_TEST_DIR, ['js']):
+        print(os.path.basename(s))
         f = open('a.js', 'w')
         f.write(open(BINARYEN_JS).read())
         if NODEJS:
             f.write(node_test_glue())
-        test_path = os.path.join(options.binaryen_test, 'binaryen.js', s)
-        test_src = open(test_path).read()
+        test_src = open(s).read()
         f.write(test_src)
         f.close()
         if MOZJS or node_has_wasm or 'WebAssembly.' not in test_src:
             cmd = [MOZJS or NODEJS, 'a.js']
-            if 'fatal' not in s:
+            if 'fatal' not in os.path.basename(s):
```

Comment:
Maybe extract a `basename` variable since its used several times in this function?

Reply:
Done